packages/medusa-file-local/src/services/local-file-service.ts

```diff
@@ -29,40 +31,91 @@ class LocalService extends AbstractFileService implements IFileService {
   async uploadFile(
     file: Express.Multer.File,
     options = {}
-  ): Promise<{ url: string }> {
-    const parsedFilename = parse(file.originalname)
+  ): Promise<{ url: string; key: string }> {
```

Comment:
suggestion(non-blocking): Create a type in `@medusajs/types`
```suggestion
  ): Promise<{ url: string; key: string }> {
```